unittests/tools/test_popeye_parser.py

```diff
@@ -0,0 +1,36 @@
+from ..dojo_test_case import DojoTestCase
+from dojo.tools.popeye.parser import PopeyeParser
+from dojo.models import Test
+
+
+class TestPopeyeParser(DojoTestCase):
+
+    def test_popeye_parser_with_no_vuln_has_no_findings(self):
+        testfile = open("unittests/scans/popeye/popeye_zero_vul.json")
+        parser = PopeyeParser()
+        findings = parser.get_findings(testfile, Test())
+        testfile.close()
+        self.assertEqual(0, len(findings))
+
+    def test_popeye_parser_with_one_warning_has_one_findings(self):
+        testfile = open("unittests/scans/popeye/popeye_one_vul.json")
+        parser = PopeyeParser()
+        findings = parser.get_findings(testfile, Test())
+        finding_title = "pods test-namespace/6cff44dc94-d92km [POP-106] No resources requests/limits defined"
+        finding_description = "**Sanitizer** : pods" + "\n\n" + \
+                            "**Resource** : test-namespace/6cff44dc94-d92km" + "\n\n" + \
+                            "**Group** : test-group" + "\n\n" + \
+                            "**Severity** : Warning" + "\n\n" + \
+                            "**Message** : [POP-106] No resources requests/limits defined"
```

Comment:
it seems `POP-106` is an id, could you try to push it in an attribute?

Reply:
@damiencarol, can you check it now?